libexec/scoop-update.ps1

```diff
@@ -57,12 +57,21 @@ if(($PSVersionTable.PSVersion.Major) -lt 5) {
 
 function update_scoop() {
     # check for git
-    if(!(Test-CommandAvailable git)) { abort "Scoop uses Git to update itself. Run 'scoop install git' and try again." }
+    if (!(Test-CommandAvailable git)) { abort "Scoop uses Git to update itself. Run 'scoop install git' and try again." }
 
-    Write-Host "Updating Scoop..."
+    $SCOOP_HOLD = get_config SCOOP_HOLD $false
     $last_update = $(last_scoop_update)
```

Comment:
```suggestion
    $last_update = last_scoop_update
```

This should work, I don't know why use `$()`

Reply:
@niheaven It is old code in files, may no one know why. 😂